package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mesg-js",
-  "version": "4.2.1",
+  "version": "4.2.2",
```

Comment:
this should be done during the release process

Reply:
```suggestion
  "version": "4.2.1",
```

Reply:
the actual version is 4.2.2 https://www.npmjs.com/package/mesg-js there is an issue with the master branch, the tag for 4.2.2 is created but the master is not updated with that